src/lib/stitching/convection/stitching.ts

```diff
@@ -32,5 +43,40 @@ export const consignmentStitchingEnvironment = (
         },
       },
     },
+
+    ConsignmentOffer: {
+      lowEstimateAmount: {
+        fragment: gql`
+          fragment ConsignmentOfferLowEstimateAmount on ConsignmentOffer {
+            currency
+            lowEstimateCents
+          }
+        `,
+        resolve: (parent, args) =>
+          amount((_) => parent.lowEstimateCents).resolve(
```

Comment:
Does this respect different currency's major/minor unit conversion ratio or does it simply [`/ 100`](https://github.com/damassi/metaphysics/blob/2e1c89f763d9f15d16c02f5ca193d5d1d641048c/src/schema/v2/fields/money.ts#L65)?

Reply:
Yup! Updated.